tests/integration/multiple_protocol_gateway/gateway/multiprotocol_gateway.py

```diff
@@ -0,0 +1,78 @@
+import grpc
+from grpc_health.v1 import health, health_pb2, health_pb2_grpc
+from grpc_reflection.v1alpha import reflection
+from pydantic import BaseModel
+from uvicorn import Config, Server
+
+from jina import Gateway, __default_host__
+from jina.proto import jina_pb2, jina_pb2_grpc
+
+
+class DummyResponseModel(BaseModel):
+    protocol: str
+
+
+class MultiProtocolGateway(Gateway):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.http_port = self.runtime_args.port[0]
+        self.grpc_port = self.runtime_args.port[1]
+        self.health_servicer = health.HealthServicer(experimental_non_blocking=True)
+
+    async def setup_http_server(self):
```

Comment:
make them private, to show what is API and what is custom?

Reply:
makes sense

Reply:
done